Source/JavaScriptCore/dfg/DFGByteCodeParser.cpp

```diff
@@ -4375,6 +4375,7 @@ bool ByteCodeParser::handleIntrinsicGetter(Operand result, SpeculatedType predic
         return false;
 
     switch (variant.intrinsic()) {
+    case DataViewByteLengthIntrinsic:
     case TypedArrayByteLengthIntrinsic: {
```

Comment:
Let's not reuse `ArrayMode` based code below since DataView is not Array. Let's add new DataViewGetByteLength and DataViewGetByteLengthAsInt52 node types and just use DataViewUse edge.